x-pack/plugins/security_solution/public/flyout/document_details/isolate_host/index.tsx

```diff
@@ -12,6 +12,8 @@ import type { DocumentDetailsIsolateHostPanelKey } from '../shared/constants/pan
 import { PanelContent } from './content';
 import { PanelHeader } from './header';
 
+// FIXME:PT Ensure our team owns this directory
```

Comment:
that is a very good point, are you making a change to the CODEOWNERS file in this PR or in a follow up one?

Reply:
no... its supposed to be done in this PR. Looks like I forgot to get to this one. I'll make the change. thanks for catching it.